src/components/card.tsx

```diff
@@ -23,72 +31,87 @@ const CardLink: FC<CardLinkProps> = ({ name, link, color }) => (
 
 type Props = {
   /**
-   * The card title (can be an Element)
+   * The card header (should include the wanted heading level)
    */
-  title?: ReactNode;
+  header?: ReactNode;
   /**
-   * The subtitle (can be an Element)
+   * Does the card header need a separator? Defaults to true
    */
-  subtitle?: ReactNode;
+  headerSeparator?: boolean;
   /**
    * Links to be displayed at the bottom of the card
    */
-  links?: Array<CardLinkProps>;
+  links?: Array<CardActionProps>;
   /**
    * Should the card styling show it as active or not
    */
+  // TODO: where is that used?
```

Comment:
This should have been used whenever a Card checkbox is selected. It looks like it's been lost, but also I think we need to think about it carefully as we have:
- mouseover
- consistency with selection in the results table
- there was mention of alternate card background colour like in results table (not in favour of that)

Reply:
I'll remove it then since the checkbox is not handled by the card

Reply:
Mmm I don't know, I'd rather have checkbox as a modifier on the card pattern, as it is included in the data table pattern and we could then share styles between them for consistency. Something else I was noticing while using the website was that I think it would be good to increase the clickable area for the Card checkbox as it's easy to miss and then navigate to wherever the card links to.

Reply:
Let's make a Jira for that for later then.
There's so much more to handle if we want to handle checkboxes in a correct way:
 - Accessibility
 - Target size
 - Who has the control of the checkbox state (uniprot-website or franklin's card?)
 - Ctrl/Cmd-click and Shift-click behaviours
 - ... and probably more

Reply:
OK - I thought the Card used to have the checkbox as part of it but it looks like it didn't. Must have happened in another refactor!

Reply:
It used to "assume" it contained a checkbox, but in its child, because there was style for that that I removed because it didn't make any difference in where we used it.